tools/README.md

```diff
@@ -118,6 +118,99 @@ The conversion script has a minimum memory requirement of 2X of the size of the
 python3 convert_feat_to_wholegraph.py --dataset-path ogbn-mag240m-2p --node-feat-names paper:feat --low-mem
 ```
 
+## Generate Mitra embeddings for graph data
+GraphStorm provides a tool to generate Mitra embeddings using AutoGluon's Mitra TFM (Tabular Foundation Model). The `gen_mitra_embedding.py` script supports both the MovieLens-100k dataset and custom parquet data. Mitra embeddings can be used as pre-trained node features to improve model performance.
+
+**Important**: The current version of Mitra TFM supports a maximum of 10 classes for multiclass classification problems.
```

Comment:
Could you specify which version of Mitra TFM, e.g., v1.*, is the `current` version?